### DISCUSSION TITLE
How do you use your panels ?

### DISCUSSION THREAD

**Discussion by rudgej** [Blackymas/NSPanel_HA_Blueprint#1206] [Q&A]
I wondered what everyone else uses the panels for ? 

It might be nice for us to share to see if there are uses other have not thought of or thought possible. 

I have 15 configured panels. The house is almost finished but only 10 are currently in use. 

In each room they act as scene controllers. The scenes are mostly selected using the hardware buttons. 

Left off. Long left hi. Right. " normal " long right medium. 

The first 1/2 pages show the lights and also blinds / curtains. They act as nice light switches really. 

Page 4 is house control. Ac off. Ac on etc. away mode. Evening extend etc which set helpers. 

I don't use chips or home page switches as I don't really see a use case. These code is set up for guest WiFi which works well. 

What do you do with yours ?

**Comment by edwardtfn** [comment]
Great idea!
I have 3 panels:

#### Office:
This panel is a US model on landscape mode, sitting over my working desk. The buttons will be controlling the ceiling lights (Nanoleaf) and the "meeting lights" (Philips Hue). Long press will give me to the detailed page of those lights, so I can adjust temperature, color and brightness.
The 2 first button pages are controlling the curtains and shutters (Ikea), the switches for my displays and docking station (a generic Zigbee power strip), etc. The climate is connected to a TRV (Tado) in the room. The media player is connected to an Alexa Echo.
Usually I use this panel for my tests, so in the other pages you will find all kind of controls, many times to virtual (fake) devices.

#### Entrance:
This is probably the most useful at home and it is installed just beside the front door. Me and my wife love to see the outside temperature when leaving home. Left button will control the light in that room (Philips Hue) and right button triggers a "leaving home" script, which will turn on alarms, warm the car, turn off lights, etc.
I use this panel  to control my alarm system, which are basically câmeras (Eufy) to watch the dogs when we are out.
This is a EU model

#### Guest room
This is an US model in portrait, used to switch the lights (Nanoleaf) with the left button or toggle the shutter/blinder (Ikea). It's also possible to set room temperature (Tado), but this is almost never used.

**Comment by rtm-soft** [comment]
Have 3 panels but will get more.

Master-bed:

2 panels on nightstands, buttons to turn hue lights on/off in bedroom, close bedroom shade. Love seeing the weather when I wake up. Will try to use it as an alarm clock. 

Office:

1 at desk so I can control brightness lights, weather info etc.

Chips show status of garage/doors, humans detected at front door etc. Ecobee thermostat status at home/rental unit. Entity list shows all temperature around the house

**Comment by bentouspam** [comment]
I have 2 panels, one on each nightstand.
would like to add an alarm clock feature and a short planning of the day.

in a future house, I would have some panels in the living room an the entry.

**Reply by edwardtfn** [reply]
Alarm clock: #1441 
Short planning of the day: #1442

**Comment by kroessig** [comment]
I would also like to take the opportunity to introduce my NSPanel and express my gratitude to the developers. 

![PXL_20231220_203529810](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/139384422/70c1c32c-6596-4eab-86ca-cef189ec4fca)
I use it in my livingroom. The normal functions should be known (QR-Code, On-Off, ButtonsPages (lights and shutter) and EntityPages ).

With one hardware button I control my Nanoleafs (wall lamp) and with the second one I switch between two scenes. 

With the custom buttons I would like to emphasise **my timer**. If this switch is activated, a **push message is displayed** on the panel after a defined time and an **rttl sound is played.** The timer is reset and is inactive again. For exsample, i use this so that I don't forget to close the windows after airing the room.

At the beginning I hardly paid any attention to the chips as I could hardly recognise them. Since they are bigger and I can **control them by colour**, I look at them constantly. 
The stored entities are usually switches that are set to "on", but the **icons are defined via states in the HA.** As an example, my "crossed out water drop" changes to a normal one when the domestic water treatment changes from off to auto. To make this even more visual, the droplet turns grey when there is no heating and turns orange when the hot water is heated.
more:
- The battery symbol and it's colour changes depending on whether the battery is being charged or discharged.  
- The waste symbol changes colour depending on which waste bin is next. Of course, a push notification also appears on the panel the evening before. 

I am currently working on the climate page, as I would like to have something different here.
next year, when my other rooms get electric shutters, a nspanel will be installed everywhere

**Reply by f0b04707** [reply]
Very nice ! How do you update the colors and the icons of the chips ? Thanks !

**Reply by kroessig** [reply]
Hey, take a look at the post.
[Link](https://github.com/Blackymas/NSPanel_HA_Blueprint/discussions/1183#discussioncomment-8209579)

**Comment by MichaelHeimann** [comment]
Hi,

I have 9 panels - one in every room which had a underfloor heating thermostat which were replaced by the nspanels.

So what do I use them for:

prio 1: thermostat.
prio 2: hardware buttons for light/radio control - basically on/off switches, partly triggering automations e.g.: radio button turns receiver on, sets input source to net_radio and sets volume to 30%; if radio is on, the same buttons turns it off.
prio 3: display time, weather and solar info

I do not use button pages at all since I think swiping is not intuitive and mobile phones are easier to achieve the same (actually better) functionality. I'd actually like to turn them off to avoid situations when someone accidentally swiped into a blank page.
